src/Filter/types/index.ts

```diff
@@ -56,7 +56,7 @@ export interface FilterModule<T> {
     /**
      * Generates the url query param value(s) for saving filter value(s) in the browser address bar (key is automatic).
      */
-    getBrowserQueryUrlValue(value: T): string | string[] | null | undefined;
+    getBrowserQueryUrlValue(value: T): unknown;
```

Comment:
Why is this now `unknown`?  Sorry, trying to better understand how the types work in this module.

Reply:
I think the return type was too restrictive and we try avoiding `any` when we can. If this were to be overridden, it would be best for the implementer to provide correct type assertions to guide it from unknown based on what they're doing in the function.

Reply:
Yeah, Jon is correct. As shown by our ability to use numbers as SelectOption values, keeping it as string was no longer appropriate. I tried doing `T | T[] | null | undefined`, but that is too restrictive as well since other filters chose to parse them completely differently as well (see AddionalJSONFilter and ListFilter implementations). Seems best to force the end user to determine what they expect the value to come back as when they try to consume the value.

Reply:
Per our discussion on how getBrowserQueryUrlValue is meant to be used, I've updated to code to force the return value to be a `string`.